lib/DDG/Spice/Symbolab.pm

```diff
@@ -0,0 +1,23 @@
+package DDG::Spice::Symbolab;
+
+use DDG::Spice;
+
+spice is_cached => 1;
+
+name "Symbolab";
+description "Get a solution to a math problem from symbolab.com";
+primary_example_queries "compute integral of x^2", "solve x+2>3x";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Symbolab.pm";
+
+spice to => 'https://www.symbolab.com/ddg?query=$1';
+spice wrap_jsonp_callback => 1;
+
+triggers startend => "calculate", "solve", "compute";
+handle remainder => sub {
+    if ($_ =~ /[\^\*\=\+\-\/\\]/ or $_ =~ /\b(sin|cos|tan|cot|csc|sec|ln|log|sqrt|integral)/i) {
```

Comment:
what about "limit" and symbols like ">", "<" etc?

I’d love for this IA to handle as many kinds of math questions as it can handle, that aren't already supported by our builtin calculator.

For example, simple equations like 1+1 or sin(1) can be handled by Calculator.pm -- there's no need for us to spend time requesting results for those from your API when they're trivial to solve.

But for complex queries involving limits, derivatives, integrals, inequalities, etc (all stuff that Calculator.pm ***can't*** handle) it makes perfect sense for use to leverage Symbolab and benefit the user by also showing them a link to Symbolab's step-by-step instructions!